workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -292,3 +292,19 @@ def db_schema_get_current_revision():
     ])
     revision = output.split(' ', 1)[0]
     return revision
+
+
+def stage_db_schema_get_current_revision():
+    """Get stage database schema revision.
+
+    :returns: Current revision
+    :rtype: str
+
+    """
+    output = subprocess.check_output([
```

Comment:
I don't know what's the right handling here, but for community edition this will not work..